pkgs/development/tools/yarn2nix-moretea/yarn2nix/lib/fixPkgAddMissingSha512.js

```diff
@@ -34,7 +34,7 @@ function getSha1(url) {
 }
 
 // Object -> Object
-async function fixPkgAddMissingSha1(pkg) {
+async function fixPkgAddMissingSha512(pkg) {
   // local dependency
 
```

Comment:
I would also add something like

```
if (pkg.integrity) {
  return pkg;
}
```

as according to my understanding, `integrity` has priority over the hash in `resolved`.